subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio_nrf5_dppi.h

```diff
@@ -100,7 +100,11 @@ static inline void hal_radio_end_time_capture_ppi_config(void)
  */
 static inline void hal_event_timer_start_ppi_config(void)
 {
+#if defined(CONFIG_BT_CTLR_NRF_GRTC)
+#else /* !CONFIG_BT_CTLR_NRF_GRTC */
```

Comment:
Was something forgotten here, or would it make sense for this to just be 
#if !defined(CONFIG_BT_CTLR_NRF_GRTC) ?